src/content/decentralization/index.md

```diff
@@ -18,57 +18,144 @@ summaryPoints:
 
-You would have to learn a lot to be able to *prove* that these systems were trustworthy. Of course, the real barrier to detailed knowledge is social. Apart from learning the technical skills, you would need to secure a management position in the company to get access to the details.
+## Decentralization in Ethereum {#ethereum-decentralized}
 
-When you look at it that way, the costs don't add up. Assuming you have zero technical knowledge, and that you need to prove that you could trust each payment processing system, here's the minimum you would need to invest:
+First things first: Nobody owns Ethereum. The [Ethereum Foundation](https://ethereum.foundation) provides support, but so do tons of other individuals and organizations.
 
-- Visa: Dedicating your career to becoming an engineering lead at Visa. 
-- Ethereum: An hour or two of [studying](/developers/docs/smart-contracts/languages/) [publicly](/developers/docs/) [available](/developers/tutorials/) [materials](/developers/learning-tools/) every day for 3-6 months.
+Building on that, Ethereum is decentralized in many other important ways. People operate nodes across the world, there are many different client software options, and it interoperates (often via third-party [decentralized apps](https://ethereum.org/en/dapps/)) with a lot of other decentralized protocols. It is built to resist spammers, to provide clear consensus, and to be unprofitable (or at least proportionately expensive) to attack. 
 
-Bonus: If you know anything at all about JavaScript or Python, you already have a huge head start.
+Ethereum is also *centralized* in a very important way — there is an agreed-upon set of rules (in the form of computer code) that every node should follow. This allows the system to operate predictably. However, it bears mentioning that these rules are upgraded and maintained in a decentralized way.
+
+### Smart contracts {#smart-contracts}
+
+Behind every dapp (decentralized app) on Ethereum, there is a smart contract. This is a program that runs on the Ethereum network for users to interact with.
+
+Smart contracts are an essential part of Ethereum's decentralized infrastructure. They can theoretically provide any service that a computer can. They can execute currency exchanges, retrieve and display information, make calculations, and so on.
+
+We say that Ethereum can solve "theoretically" any problem because there are some limits: namely the expensive digital real estate in the [Ethereum Virtual Machine](https://ethereum.org/en/glossary/#evm) and on the [blockchain](https://ethereum.org/en/glossary/#blockchain). 
+
+## Pros and cons of using Ethereum {#pros-cons}
```

Comment:
I think this needs to directly reference Ethereum as a decentralized platform...

Ethereum's principles give incredible importance to decentralization. This has some amazing pros but also some cons, or behaviour that you probably won't be used to as a user of the internet today...